### ISSUE TITLE
Issue with dragging card to another list in RTL layout

### ISSUE THREAD

**Issue by ahangarha** [stefan-niedermann/nextcloud-deck#512]
**Describe the bug**
When the layout of app is RTL (due to system language being set to some RTL languages such as Persian, Arabic, Urdu,...) there is an issue with dragging card to other lists. If user want to drag a card into the list on right side, the user should drag it to left (and same for opposite direction)

**To Reproduce**
1. Set system language to some RTL languages such as Persian, Arabic, Urdu,...
2. try to drag a card to either of lists on left or right side of the current list

**Expected behavior**
I should drag the card to left for moving it the list on left side and drag it to right if I wan to move it to the list on right.

**Smartphone (please complete the following information):**
- Device: Moto G5s Plus
- Android-Version: 8.1.0
- App-Store:
  - [ ] Google Play Store
  - [x] F-Droid

**Comment by stefan-niedermann** [comment]
Will be fixed in the next version, thank you for report!

**Comment by desperateCoder** [comment]
@ahangarha thanks for the reports! Personally, I don't have a clue how it should behave RTL, so your issues are pure gold. As @stefan-niedermann already said, please keep complaining, if you see that something is wrong!

**Comment by stefan-niedermann** [comment]
@desperateCoder you can enable "Force RTL layout for all languages" in android developer options, also on emulated devices.

Something like the drag and drop issue really seems to be obvious 😆 but honestly i don't have any idea about how icons and graphics should behave, or tabs^^ So as @desperateCode said: keep complaining!

**Comment by ahangarha** [comment]
Thank you so much for your quick respond and fix.

I am proudly using Nextcloud and recently I saw massive improvement on Deck so I decided to use it seriously. And as Iranian who is concerned about localization, I am focusing on RTL issues. So I will bother you again and again :D

I wish I could help with coding as well but reporting issues and translating into Persian as well as talking to public about it is what I can do.

All the best
